src/fides/api/schemas/connection_configuration/connection_secrets_mysql.py

```diff
@@ -8,6 +9,12 @@
 )
 
 
+class MySQLSslMode(str, Enum):
```

Comment:
nit: we should upper case all acronyms
> Note: When using acronyms in CapWords, capitalize all the letters of the acronym. Thus HTTPServerError is better than HttpServerError.

https://peps.python.org/pep-0008/#descriptive-naming-styles
```suggestion
class MySQLSSLMode(str, Enum):
```